configs/base/index.js

```diff
@@ -176,14 +187,35 @@ module.exports = {
         },
         allowList: {
           attr: true,
-          argTypes: true
+          argTypes: true,
         },
         checkProperties: true,
       },
     ],
     "unicorn/throw-new-error": "error",
-    // prettier handles these
-    "object-curly-newline": "off",
-    "max-len": "off",
+    "prettier/prettier": [
+      "error",
+      {
+        printWidth: 100,
+        tabWidth: 2,
+        singleQuote: true,
+        trailingComma: "all",
+        proseWrap: "always",
+        overrides: [
+          {
+            files: "*.json",
+            options: {
+              printWidth: 999999,
```

Comment:
Does this value means `json` files will be written in a single line?

Reply:
this means that they _can_ be written in a single line, which now thinking about it does not make a lot of sense.

Reply:
It might have been a leftover from an older version of prettier where long values where wrapped to the next line.

so:
```json
{
  "key": "value .............. very long ..............."
}
```
would become
```json
{
  "key":
    "value .............. very long ..............."
}
```

Just tested in the REPL, it doesn't do that anymore. So I think we can remove it.

OR, it was copied over from one of the projects where we generated mock files that were API responses (without formatting), and we didn't want prettier to constantly handle those. Either way, we can remove it.